pkg/chubaofs/nodeserver.go

```diff
@@ -234,3 +245,158 @@ func nodeGetVolumeStats(_ context.Context, volumePath string) (*csi.NodeGetVolum
 		},
 	}, nil
 }
+
+// getAttachedPVOnNode finds all persistent volume objects attached in the node and controlled by me.
+func (ns *nodeServer) getAttachedPVOnNode(nodeName string) ([]*v1.PersistentVolume, error) {
+	vaList, err := ns.Driver.ClientSet.StorageV1().VolumeAttachments().List(metav1.ListOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("unable to list VolumeAttachments: %v", err)
+	}
+
+	nodePVNames := make(map[string]struct{})
+	for _, va := range vaList.Items {
+		if va.Spec.NodeName != nodeName {
+			continue
+		}
+		if va.Spec.Attacher != DriverName {
+			continue
+		}
+		if !va.Status.Attached {
+			continue
+		}
+		if va.Spec.Source.PersistentVolumeName == nil {
+			continue
+		}
```

Comment:
you can simplify with:
```suggestion
if va.Spec.NodeName == nodeName && va.Spec.Attacher == DriverName && va.Status.Attached && va.Spec.Source.PersistentVolumeName != nil {
   nodePVNames[*va.Spec.Source.PersistentVolumeName] = struct{}{}
}
```

Reply:
done.